packages/run-protocol/src/reserve/assetReserve.js

```diff
@@ -118,6 +119,36 @@ const start = async (zcf, privateArgs) => {
   const makeAddCollateralInvitation = () =>
     zcf.makeInvitation(addCollateralHook, 'Add Collateral');
 
+  const { brand: runBrand } = await E(runMint).getIssuerRecord();
+  const { publication: metricsPublication, subscription: metricsSubscription } =
+    makeSubscriptionKit();
+
+  // shortfall in Vaults due to liquidations less than debt. This value can be
+  // reduced by various actions which burn IST.
+  let vaultShortfall = AmountMath.makeEmpty(runBrand);
+
+  const updateMetrics = () => {
+    const metrics = harden({
+      allocations: getAllocations(),
+      shortfall: vaultShortfall,
```

Comment:
for naming, `shortfall` sounds like an event metric but this is a sum. consider `totalShortfall`